### DISCUSSION TITLE
AstroDB, export alias from drizzle

### DISCUSSION THREAD

**Discussion by NickDubelman** [withastro/roadmap#889] [Proposal]
# Summary

Should be able to `import { alias } from 'astro:db'`

# Background & Motivation

I just hit a situation where I needed to join on the same table twice in the same query (sports-themed app where there is a home team and an away team).

I consulted the Drizzle docs and found `alias`. In order to use it, I had to add drizzle itself. I checked out the types exported by `astro:db` and I see things like `eq`, `gt`, etc. It seems like `alias` belongs in these set of exports that are automatically included from Drizzle.

# Goals

- Slightly reduce the set of cases where devs need to add Drizzle itself

# Example

This is what I did to get alias:
```
pnpm i drizzle-orm
import { alias } from 'drizzle-orm/sqlite-core'
```

I'd prefer to:
```
import { db, alias, ... } from 'astro:db'
```

**Comment by matthewp** [comment]
Sounds good to me, if you want to submit a PR I think we could definitely accept it. thanks!

**Reply by NickDubelman** [reply]
Okay, I [submitted a PR](https://github.com/withastro/astro/pull/10789).. sorry if I missed anything!
